src/admin/vote/Assign.jsx

```
@@ -1,5 +1,5 @@
 import { collection, doc, getDoc, getDocs, setDoc } from "firebase/firestore";
-import { confirm, snackbar } from "mdui";
+import { confirm, prompt, snackbar } from "mdui";
```

ESLint

[error] 'prompt' is defined but never used.

(no-unused-vars)